src/datachain/cli/__init__.py

```diff
@@ -161,6 +161,7 @@ def handle_dataset_command(args, catalog):
             all=args.all,
             team=args.team,
             latest_only=not args.versions,
+            dataset_name=args.dataset_name,
```

Comment:
let's make it consistent with `rm`, `edit` (name vs dataset_name` (probably consider later updating pull?) - naming should be consistent and concise